datamade/how-to
# Migrate and revise environment setup, update README index

## Overview

See description.

In addition to adding Docker, I did some finessing of the language in our environment setup and also added GatsbyJS to the section on static sites.

I also made the editorial decision to alphabetize the contents in the README to make things easier to find.

Handles https://github.com/datamade/ops/issues/634.

## Testing Instructions

* View the rendered [README.md](https://github.com/datamade/how-to/blob/hec/onboarding-revision/README.md) and [environment-setup.md](https://github.com/datamade/how-to/blob/hec/onboarding-revision/environment-setup.md) files, and confirm the markdown looks nice.
 * Click links and confirm they resolve.
 * Suggest any changes (please and thank you)!
